platform/centec/sonic-platform-modules-e582/48x2q4z/modules/dal_kernel.c

```diff
@@ -580,25 +580,6 @@ _dal_set_msi_enabe(unsigned int lchip, unsigned int irq_num)
         msi_irq_base[lchip] = dal_dev[lchip].pci_dev->irq;
         msi_irq_num[lchip] = 1;
     }
-    else
-    {
```

Comment:
it is better to de-dup this dal_kernel.c I can see you have them in three places. since it is a platform driver. better to put in platform/centec and then install it on all centec platforms.